core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -721,6 +784,103 @@ private ExprValue exprWeek(ExprValue date, ExprValue mode) {
         CalendarLookup.getWeekNumber(mode.integerValue(), date.dateValue()));
   }
 
+  private ExprValue unixTimeStamp() {
+    return new ExprLongValue(Instant.now().getEpochSecond());
```

Comment:
Should this depend on `now()` implementation in https://github.com/opensearch-project/sql/pull/754?

Reply:
How can I do this properly for `unix_timestamp()` without arguments only?
Or I can cache all `unix_timestamp` calls.

Reply:
Sorry, I just finished that PR and may misunderstood. In that PR, we treated `now()` as constant function which is replaced by a "cached" constant value. So no matter how many call to now(), they are replaced by same literal value during analysis. Ignore my previous comment if this is not the case for unix_timestamp().